src/shared/components/InputBase/InputBase.styles.ts

```diff
@@ -1,101 +1,67 @@
 import { colors, sizes, transitions, typography } from '@/shared/theme'
-import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
 type FormGroupProps = {
   error?: boolean
   disabled?: boolean
 }
 
-export const FormGroup = styled.div<FormGroupProps>`
-  position: relative;
+export const FormGroup = styled.label<FormGroupProps>`
+  display: inline-block;
+  width: 100%;
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'auto')};
+  opacity: ${({ disabled }) => (disabled ? 0.5 : 1)};
+
+  input:-webkit-autofill,
+  input:-webkit-autofill:hover,
+  input:-webkit-autofill:focus,
+  textarea:-webkit-autofill,
+  textarea:-webkit-autofill:hover,
+  textarea:-webkit-autofill:focus {
+    // This is madness, but basically, it will make autofill transparent for a long time.
+    // Changing background-color and color won't work
+    transition: color 99999s, background-color 99999s;
+    transition-delay: 99999s;
+  }
```

Comment:
This looks error-prone. Do we really need this, are the autofill colors that bad?

Reply:
Maybe, it's not that bad after all. 
![image](https://user-images.githubusercontent.com/51168865/113710115-9b2b9780-96e3-11eb-9e21-f4878bcbe675.png)

I'll remove this.